pkg/operator/operator.go

```diff
@@ -111,18 +121,16 @@ func NewOperator(opts Options) Operator {
 			interval:          opts.WatchdogInterval,
 			maxRestartsPerMin: opts.WatchdogMaxRestartsPerMin,
 		}
-		err = mgr.Add(wd)
```

Comment:
Nit, but I very much prefer not condensing the error into the `if`. More readable, and avoids allocating a new variable for holding the error :)